### ISSUE TITLE
can't install  0.13.13 with rpm

### ISSUE THREAD

**Issue by rparree** [sbt/sbt-launcher-package#126]
I am trying to upgrade to  0.13.13 on a f24 machine, but nothing is being updated.

The repo has as `baseurl=http://dl.bintray.com/sbt/rpm`

Performing a `sudo dnf list available sbt --showduplicates` gives:

```
sbt.noarch                                                          0.12.4-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.0-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.1-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.1-8.fc24.1                                                          fedora           
sbt.noarch                                                          0.13.2-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.5-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.5.1-1                                                               bintray--sbt-rpm 
sbt.noarch                                                          0.13.6-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.7-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.8-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.9-1                                                                 bintray--sbt-rpm 
sbt.noarch                                                          0.13.12-1                                                                @bintray--sbt-rpm
sbt.noarch                                                          0.13.12-1                                                                bintray--sbt-rpm 
```

**Comment by eed3si9n** [comment]
I see sbt-0.13.13.rpm there. Could you try again please?

**Comment by manub** [comment]
I have the same problem - I've found that `yum install sbt-launcher-packaging` seem to be working.

```
[root@5811e2ac2395 /]# yum install sbt-launcher-packaging
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * centos-7-extras: os-rpm-repo-tcs.int.klarna.net
 * centos-7-os: os-rpm-repo-tcs.int.klarna.net
 * centos-7-updates: os-rpm-repo-tcs.int.klarna.net
 * epel-7: os-rpm-repo-tcs.int.klarna.net
 * klarna-ca: os-rpm-repo-tcs.int.klarna.net
Resolving Dependencies
--> Running transaction check
---> Package sbt-launcher-packaging.noarch 0:0.13.13-1 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

============================================================================================================================================================================================================================================================================================================================================================================
 Package                                                                                           Arch                                                                              Version                                                                              Repository                                                                                   Size
============================================================================================================================================================================================================================================================================================================================================================================
Installing:
 sbt-launcher-packaging                                                                            noarch                                                                            0.13.13-1                                                                            bintray--sbt-rpm                                                                            1.0 M

Transaction Summary
============================================================================================================================================================================================================================================================================================================================================================================
Install  1 Package

Total size: 1.0 M
Installed size: 1.2 M
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : sbt-launcher-packaging-0.13.13-1.noarch                                                                                                                                                                                                                                                                                                                  1/1
  Verifying  : sbt-launcher-packaging-0.13.13-1.noarch                                                                                                                                                                                                                                                                                                                  1/1

Installed:
  sbt-launcher-packaging.noarch 0:0.13.13-1

Complete!
[root@5811e2ac2395 /]# sbt --version
Getting org.scala-sbt sbt 0.13.13 ...
```

**Comment by jonathandbyrd** [comment]
i had exact problem, even though i can see the .13.13 release as latest; for whatever reason, it was puling .13.12.

i followed what manub did and got an error on a dependency conflict with 13.12 so i "yum remove" and then installed using command manub cited and it worked.

**Comment by rparree** [comment]
Can also confirm that the solution from @manub works.

**Comment by eed3si9n** [comment]
It seems like we somehow messed up the package name then.

**Comment by eed3si9n** [comment]
Since 0.13.13 was published on 2016-10-28 and a few days later we patched https://github.com/sbt/sbt-launcher-package/pull/127, so I'm guessing that's now fixed.

**Comment by eed3si9n** [comment]
I've published sbt-0.13.13.1.rpm using the correct package name: sbt.

On EC2 I've confirmed that the following works:

```
$ curl https://bintray.com/sbt/rpm/rpm | sudo tee /etc/yum.repos.d/bintray-sbt-rpm.repo
....
$ sudo yum install sbt
....
Total download size: 46 M
Installed size: 153 M
Is this ok [y/d/N]: y
....
Installed:
  sbt.noarch 0:0.13.13.1-1
....
$ sbt new scala/scala-seed.g8
Getting org.scala-sbt sbt 0.13.13 ...
....

Minimum Scala build.

name [My Something Project]: hello

Template applied in ./hello

$ cd ./hello/
$ sbt run
....
[info] Done updating.
[info] Compiling 1 Scala source to /home/ec2-user/hello/target/scala-2.12/classes...
[info] 'compiler-interface' not yet compiled for Scala 2.12.1. Compiling...
[info]   Compilation completed in 21.297 s
[info] Running example.Hello
hello
[success] Total time: 28 s, completed Jan 4, 2017 9:59:42 PM
```

Sorry it took me a while to realize what's going on.
